src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -346,15 +346,33 @@ export class MeetingManager implements AudioVideoObserver {
     );
   }
 
-  async listAndSelectDevices(): Promise<void> {
+  async listAndSelectDevices(deviceLabels: DeviceLabels | DeviceLabelTrigger = DeviceLabels.AudioAndVideo): Promise<void> {
     await this.updateDeviceLists();
 
-    const hasAudioInput = this.audioInputDevices?.some(value => value.label) || false;
-    const hasAudioOutput = this.audioOutputDevices?.some(value => value.label) || false;
-    const hasVideoInput = this.videoInputDevices?.some(value => value.label) || false;
+    // If `deviceLabels` is of `DeviceLabelTrigger` type, no device will be selected.
+    // In this case, you need to handle the device selection yourself.
+    if (typeof deviceLabels === 'function') return;
+
+    let isAudioDevicesRequested: boolean = false;
+    let isVideoDevicesRequested: boolean = false;
```

Comment:
Nit: you can un-plural these (`isAudioDeviceRequested`).

Reply:
Makes sense. Updated them to singular form. Thank you for your help.